src/NUnitConsole/nunit3-console.tests/ConsoleRunnerTests.cs

```diff
@@ -11,10 +11,15 @@ namespace NUnit.ConsoleRunner.Tests
 {
     class ConsoleRunnerTests
     {
-        [Test]
+        [Test, Ignore("Needs to be rewritten")]
```

Comment:
Ideally, they should be telling you. That's why we made the `Reason` field required. But I agree you may not know. Anyhow... slightly separate from this issue.